src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -246,17 +245,8 @@ def validatePhEDExSubscription(arguments):
     for site in arguments.get("AutoApproveSubscriptionSites", []):
         if site.endswith('_MSS'):
             raise WMSpecFactoryException("Auto-approval to MSS endpoint is not allowed: %s" % site)
-    if arguments.get("SubscriptionPriority", "Low").lower() not in PhEDEx_VALID_SUBSCRIPTION_PRIORITIES:
+    if arguments.get("SubscriptionPriority", "Low").lower() not in RUCIO_RULES_PRIORITY.keys():
```

Comment:
Same comment as above, there is no need to explicitly ask for the dictionary keys, so only `not in RUCIO_RULES_PRIORITY` would be enough.

Reply:
Done with https://github.com/dmwm/WMCore/pull/11181/commits/edb3ee5780ddf03634821fa190f0a35db9d51895